patches/common/chromium/.patches.yaml

```diff
@@ -322,3 +322,8 @@ patches:
     https://chromium.googlesource.com/chromium/src/+/f47b361887a31cccf42a6e21a82bccf28372bdaa%5E%21
     In the long term we should investigate why it happened, and take a more
     formal fix. But for now I'm just make this assertion silently pass away.
+
+  owners: torycl
```

Comment:
There should be `- ` on the previous line to indicate a new list item:
```yaml
-
  owners: torycl
```

Reply:
Done